sdk-base/src/main/java/com/mapbox/maps/plugin/locationcomponent/LocationComponentPlugin.kt

```diff
@@ -53,6 +53,20 @@ interface LocationComponentPlugin :
    */
   fun removeOnIndicatorBearingChangedListener(listener: OnIndicatorBearingChangedListener)
 
+  /**
+   * Adds a listener that gets invoked when indicator accuracy radius changes.
+   *
+   * @param listener Listener that gets invoked when indicator accuracy radius changes
+   */
+  fun addOnIndicatorAccuracyRadiusChangedListener(listener: OnIndicatorAccuracyRadiusChangedListener)
+
+  /**
+   * Removes a listener that gets invoked when indicator accuracy radius changes.
+   *
+   * @param listener Listener that gets invoked when indicator accuracy radius changes.
+   */
+  fun removeOnIndicatorAccuracyRadiusChangedListener(listener: OnIndicatorAccuracyRadiusChangedListener)
+
```

Comment:
Should we consider adding methods to this interface breaking change?

Reply:
We allow users to create their own plugin implementation, this could be a breaking change if they implemented their own location component plugin.

Reply:
cc @mapbox/maps-android

Reply:
Good catch @pengdev. 
BTW shouldn't our CI should have catched that? @Chaoba I remember you were working with that script, do you have an idea why it did not catch it?

About this particular idea - guess we could add those methods to our impl class only..

Reply:
The check output for said it's a patch release, need to revisit what happened.
`Semver-check TAGGED_RELEASE_VERSION:, LAST_VERSION: 10.2.0-beta.1, API_COMPAT_LEVEL: patch
`

Reply:
maybe we can add this to a separate interface specific to accuracy radius functionality?

Reply:
With current design Alternatively we could add extension functions to the interfaces, and within the implementation, cast the interface to actual implementation and use the new function on the implementation class, something like:

```
fun LocationComponent.addAccuracyListener(..) {
  (this as LocationComponentImpl).addAccuracyListener(..)
}
```